Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ViewExtensions/ViewExtensionsCode.bind

```diff
@@ -0,0 +1,22 @@
+<Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.StatusBarExtensionsCode"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:local="using:Microsoft.Toolkit.Uwp.SampleApp.SamplePages"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      xmlns:viewHelper="using:Microsoft.Toolkit.Uwp.UI.Extensions"
```

Comment:
can we change the namespace declaration to viewExtensions or just extensions. "viewHelper" doesn't make sense for the namespace